.github/workflows/release.yml

```diff
@@ -15,12 +15,15 @@ jobs:
         with:
           java-version: '17'
           distribution: 'adopt'
-          cache: maven
-      - name: Build with Maven
-        run: mvn --batch-mode --update-snapshots package
+      - name: Validate Gradle wrapper
+        uses: gradle/wrapper-validation-action@v1
+      - name: Build with Gradle
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: build
       - name: Release with Notes
         uses: softprops/action-gh-release@v1
         with:
-          files: target/VoxelSniper*.jar
+          files: output/VoxelSniperBukkit*.jar
```

Comment:
Best to rename this to voxelsniper-bukkit-{version}. no caps and no Camel case

Reply:
https://github.com/KevinDaGame/VoxelSniper-Reimagined/pull/51/commits/f96d57bce9c9008b31c6de3254f098a0392572c5